src/services/expenses.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+let expenses = [];
+let newExpenceId = 1;
```

Comment:
Check what ids you would have. Better use Math.random multiply with new Date inside of the function where you use it.